README.md

```diff
@@ -36,47 +40,95 @@ Navigate your browser to instance address, you will see home page with usage ins
 
 On dev env you may want to enable proxy to allow to create subdomains and tests composer organizations:
 
-```
+```bash
 composer proxy-setup
 ```
 
 This will create `repman.wip` domain. Then you can add other domains with:
 
-```
+```bash
 symfony proxy:domain:attach your-organization.repman
 ```
 
 ### CLI commands
 
- - `bin/console repman:metadata:clear-cache` - clear packages metadata cache (json files)
-
-
-## Roadmap
-
- - [ ] support for docker (to allow to create repman instance with docker)
- - [ ] manual webook installation
+- `bin/console repman:metadata:clear-cache` - clear packages metadata cache (json files)
 
 ## Integration
 
 Callbacks:
- - `/auth/{provider}/check`
- - `/register/{provider}/check`
- - `/user/token/{provider}/check`
+
+- `/auth/{provider}/check`
+- `/register/{provider}/check`
+- `/user/token/{provider}/check`
 
 ### GitHub
 
 Scopes:
- - registration: `user:email`
- - repositories: `read:org`, `repo`
+
+- registration: `user:email`
+- repositories: `read:org`, `repo`
 
 ### GitLab
 
 Scopes:
- - registration: `read_user`
- - repositories: `api`
+
+- registration: `read_user`
+- repositories: `api`
 
 ### Bitbucket
 
 Scopes:
- - registration: `email`
- - repositories: `repository`, `webhook`
+
+- registration: `email`
+- repositories: `repository`, `webhook`
+
+## Docker
+
+- Override with `docker-compose.override.yml` if needed.
+- Prepare your `.env.local`:
+
+```bash
+APP_HOST=repman.wip
+DATABASE_URL=postgresql://main:main@database:5432/main?serverVersion=11&charset=utf8
```

Comment:
let's move this to compose file

Reply:
all docker related envs moved to `.env.docker`